src/layouts/ProfileLayout/ProfileLayout.tsx

```diff
@@ -14,6 +15,7 @@ const ProfileLayout: FC = () => (
     </NavigationTogglerProvider>
     <main>
       <Outlet />
+      <CookiesToaster />
```

Comment:
Could you reply to this comment with a link to the technical requirement for showing a toaster on profile page?